src/main/java/com/mxf/rest/RestFullServiceApplication.java

```diff
@@ -1,9 +1,14 @@
 package com.mxf.rest;
```

Comment:
what means `mxf`? it project name? On the project name I see `rest_soft_bi_fst`, so what means `mxf`?